src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -43,8 +43,6 @@ private static IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundEventAccessOperation((BoundEventAccess)boundNode);
                 case BoundKind.EventAssignmentOperator:
                     return CreateBoundEventAssignmentOperatorOperation((BoundEventAssignmentOperator)boundNode);
-                case BoundKind.DelegateCreationExpression:
```

Comment:
Thank you - yes we decided to switch BoundDelegateCreationExpression to have OperationKind.None until we design the correct API for it.